kinto_http/batch.py

```diff
@@ -45,18 +50,35 @@ def _build_requests(self):
     def send(self):
         result = []
         requests = self._build_requests()
+        id_request = 0
         for chunk in utils.chunks(requests, self.batch_max_requests):
             kwargs = dict(method='POST',
                           endpoint=self.endpoints.get('batch'),
                           payload={'requests': chunk})
             resp, headers = self.session.request(**kwargs)
             for i, response in enumerate(resp['responses']):
```

Comment:
we could enumerate chunks and have `id_request = i * self.batch_max_requests + j` ?